gemstone/src/swapper/models.rs

```diff
@@ -195,6 +194,7 @@ pub struct SwapQuoteData {
     pub to: String,
     pub value: String,
     pub data: String,
+    pub approvals: Vec<ApprovalType>,
```

Comment:
```suggestion
    pub approvals: Vec<ApprovalData>,
```

Name suggestion or just `Approval` ?